e2e/__tests__/viteIntegration.test.ts

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
We should have e2e test cases that demonstrate the options that are supported in phase 1